content/blog/2016/2016-09-02-jenkins-world-speaker-blog-pipeline-model-definition.adoc

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Introducing a New Way to Define Jenkins Pipelines"
+tags:
+- event
+- JenkinsWorld
+author: abayer
+---
+
+NOTE: This is a guest post by Jenkins World speaker Andrew Bayer, Jenkins
+developer at link:https://www.cloudbees.com/[CloudBees].
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World, role=right]
+
+Over the last couple years, Pipeline as code has very much become the future of
+Jenkins - in fact, at this point, I'd say it's pretty well established as the
+*present* of Jenkins. But that doesn't mean it's done, let alone that it's
+perfect. While many developers enjoy the power and control that they get from
+writing Pipelines using scripting, not everyone feels the same way. A lot of
+developers want to specify their build as configuration and get on with building
+software.
+
+Pipeline scripts haven't been a good way to do that...until now.
+
+With new changes to Jenkins Pipeline, you are now able to define your Pipeline
+from configuration in your `Jenkinsfile` by installing the new
+link:https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Model+Definition+Plugin[Pipeline Model Definition]
+plugin. It's available today for you to try via the update center.
+Be sure to check the link:https://github.com/jenkinsci/pipeline-model-definition-plugin/wiki[documentation] for examples on how to get started for a
+variety of languages and platforms.
+
+Here's a quick example from link:https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/master/Jenkinsfile[the plugin's own Jenkinsfile]:
+
+[source,groovy]
+----
+pipeline {
+    tools {
+        maven "Maven 3.3.9"
+        jdk "Oracle JDK 8u40"
+    }
+
+    image label:"any-executor"
+
+    stages {
+        stage("build") {
+            sh 'echo "path: ${PATH}"'
+            sh 'echo "M2_HOME: ${M2_HOME}"'
+            sh 'mvn clean install -Dmaven.test.failure.ignore=true'
+        }
+    }
+
+    postBuild {
+        always {
+            archive "target/**/*"
+            junit 'target/surefire-reports/*.xml'
+        }
+    }
+}
+----
+
+It's still early days for this feature, with a lot of further functionality
```

Comment:
Actually...hrm. The last paragraph makes sense as a closer since it's got the reference to the talk/demo/timing, but yeah, its opening sentence isn't right.